config/database.yml

```diff
@@ -0,0 +1,24 @@
+default: &default
+  adapter: postgis
+  pool: 5
+  timeout: 5000
+  schema_search_path: "public,postgis"
+  script_dir: /usr/local/share/postgis/
+  encoding: unicode
+  username: safecast
+  password:
+
+development:
+  <<: *default
+  database: ingest-solarcast_development
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: ingest-solarcast_test
+
+production:
+  <<: *default
+  database: ingest-solarcast_production
```

Comment:
It seems to be supported, so I added configs in bce090288de0870c9b403595a4dda816abf5aa2c